tools/reccomp/reccomp.py

```diff
@@ -1,66 +1,44 @@
 #!/usr/bin/env python3
 
+import argparse
 from capstone import *
 import difflib
 import struct
 import subprocess
 import os
 import sys
 
-def print_usage():
-  print('Usage: %s [options] <original-binary> <recompiled-binary> <recompiled-pdb> <decomp-dir>\n' % sys.argv[0])
-  print('\t-v, --verbose <offset>\t\t\tPrint assembly diff for specific function (original file\'s offset)')
-  print('\t-h, --html <output-file>\t\t\tGenerate searchable HTML summary of status and diffs')
-  sys.exit(1)
+parser = argparse.ArgumentParser(allow_abbrev=False)
+parser.add_argument('original', metavar='original-binary', help='An original LEGO\N{Copyright Sign} binary')
+parser.add_argument('recompiled', metavar='recompiled-binary', help='The recompiled binary')
+parser.add_argument('pdb', metavar='recompiled-pdb', help='The PDB of the recompiled binary')
+parser.add_argument('decomp_dir', metavar='decomp-dir-pdb', help='The recompiled source tree')
```

Comment:
Some of these descriptions are fairly misleading.

- With `An original LEGO\N{Copyright Sign} binary`, it would be preferable not to use the LEGO name because this script is fairly generic and could be used with any executable decompiled/recompiled in this way. 
- `The recompiled source tree`, the source tree isn't "recompiled", it's "decompiled".
- `decomp-dir-pdb` where does "pdb" come into this?

Reply:
- No more `LEGO` anymore in the script
- Fixed the other 2 strings